Packs/FortiSIEM/IncidentFields/incidentfield-Incident_Report_Device_Name.json

```diff
@@ -0,0 +1,29 @@
+{
```

Comment:
@OfekMQmasters Either add a prefix to id, name and cliname or use common field `devicename` instead.

Reply:
Done